ios/MembraneWebRTCModule.swift

```diff
@@ -104,7 +104,7 @@ public class MembraneWebRTCModule: Module {
       try membraneWebRTC.toggleMicrophone()
     }
     
-    Property("IsCameraOn") {
+    Property("isCameraOn") {
```

Comment:
Why here camelCase and in android PascalCase?

Reply:
https://github.com/jellyfish-dev/react-native-membrane-webrtc/blob/2184aa99fd8bf36207b9e4d31e66a4b3fc56badb/android/src/main/java/com/reactnativemembrane/MembraneWebRTCModule.kt#L144

i belive that on android camelCase is used